examples/repr.md

```diff
@@ -7,9 +7,9 @@
 >>> from hubmap_api_py_client import Client
 >>> client = Client(environ['API_ENDPOINT'])
 >>> client
```

Comment:
I'd suggest getting rid of this test, so that you will use this test against devel enviroments, and never again have to dig out of this hole.